### DISCUSSION TITLE
Chinese website is not updated here.

### DISCUSSION THREAD

**Discussion by chenzhengWin** [vercel/next.js#33206] [Ideas]
Chinese website is not updated here.
![image](https://user-images.githubusercontent.com/48211880/149090952-ffa5e662-f7f6-4aa0-957e-04d5aeb2b743.png)
The path is wrong. And the right path is "**https://github.com/vercel/next-learn/tree/master/basics/learn-starter**".
Please refer to the official documentation for details, the website is"https://nextjs.org/learn/basics/create-nextjs-app/setup".
![image](https://user-images.githubusercontent.com/48211880/149091503-a9c7870b-4918-4306-bc99-854ec89b38f0.png)

**Comment by utakotoba** [comment]
In fact, the Chinese website (nextjs.cn) is not under the control of the official team, besides, the documentation for that site is stuck at version 9, and there is no update for a long time.
